.github/pull_request_template.md

```diff
@@ -14,6 +14,9 @@ Relates to #
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Other (security config, docs update, etc)
 
+## Steps to Validate
+- If this PR introduces new functionality to UDS Core, please document the steps to test the changes.
```

Comment:
```suggestion
## Steps to Validate
- If this PR introduces new functionality to UDS Core or addresses a bug, please document the steps to test the changes.
```

Think we should also note this for bug fixes, wdyt? (feel free to reword my wording)